packages/jobs/lib/execution/operations/utils/errors.ts

```diff
@@ -0,0 +1,27 @@
+import { NangoError, deserializeNangoError } from '@nangohq/shared';
+import { stringifyError } from '@nangohq/utils';
+
+export function getFormattedScriptError({ err, defaultErrorType, scriptName }: { err: unknown; defaultErrorType: string; scriptName: string }) {
+    if (typeof err !== 'object') {
+        return new NangoError('invalid_error', err);
+    }
+    if (!err) {
+        return new NangoError('invalid_error', null as any);
+    }
```

Comment:
I wish typescript had a `something` type representing an existing value that could not be null or undefined